pkg/dom0-ztools/Dockerfile

```diff
@@ -42,3 +43,13 @@ FROM scratch
 COPY --from=zfs /out/ /
 # hadolint ignore=DL3020
 ADD rootfs/ /
+RUN touch /etc/group && touch /etc/passwd \
+    && addgroup -g 0 root \
+    && adduser -S -D -H -h /root -s /bin/sh -G root -u 0 root
+### Create system-wide groups and users ###
+# /dev/tpm* is accisable to non-root users via tpms group
+RUN addgroup tpms
+# setup vtpm user and group
+RUN addgroup vtpm \
+    && adduser -D -H -h /none -s /bin/false -G vtpm vtpm \
+    && addgroup vtpm tpms
```

Comment:
```suggestion
# setup vtpm/tpms user and group
RUN addgroup -S vtpm
RUN addgroup -S tpms
RUN adduser -D -H -h /none -s /bin/false -G vtpm vtpm
RUN addgroup vtpm tpms
```

this way the `vtpm` uid is the same as it's gid

Reply:
I want to keep vtpm/tpms separate entities (hence separately commented), because I want to add more user/groups. "vtpms" is not just fore vtpm, pillar, recovertpm, etc also use tpm devices.  

and for the `-S`, AFAIK it just lets you to have a  <999 UID/GID, which busybox code seems to [confirm it](https://github.com/mirror/busybox/blob/2d4a3d9e6c1493a9520b907e07a41aca90cdfd94/loginutils/adduser.c).

Reply:
> this way the vtpm uid is the same as it's gid
you mean changing the order?

Reply:
`-S` because it is a system group and not a group of people